data/src/main/java/io/gnosis/data/repositories/TransactionRepository.kt

```diff
@@ -227,13 +227,16 @@ class TransactionRepository(
     private fun Long.toDate(): Date = Date(this)
 }
 
-fun List<ParamsDto>?.getValueByName(name: String): String? {
-    this?.map {
-        if (it.name == name && it.value is String) {
-            return (it.value as String)
-        }
-    }
-    return null
+fun List<ParamDto>?.getAddressValueByName(name: String): Solidity.Address? {
```

Comment:
Would get rid of the `?` for the extension function as it doesn't add value, AFAICT. Some for the method below 👇

Reply:
we would have to use nullablility checks everywhere we use this function